wing.py

```diff
@@ -344,11 +344,8 @@ def setup(self, **kwargs):
                 taper >= 0.15, # TODO
 
                 # Fuel volume [TASOPT doc]
-##                TCS([Afuel <= wwn*0.92*tau]),
-                # GP approx of the signomial constraint:
-
-                #.3026 computed from TASOPT 737 case, assumes all lengths in m
-                Vfuel <= 0.3026*mac**2*b*tau,
+                TCS([Afuel <= wwn*0.92*tau]),
+                Vfuel <= croot**2*b/6*(1+taper+taper**2)*cosL, # [SP]
                 WfuelWing <= rhofuel*Vfuel*g,
```

Comment:
I'm fairly certain the reason is we don't have the same semi-discretized wing as TASOPT with the taper breaks so straight implementation of the TASOPT fuel volume constraints results in wing over sizing and heavy aircraft. The constraint I developed is tuned such that it works perfectly for an airplane that is the same exact size as TASOPT output.